bin/bump.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env sh
```

Comment:
Can we change the file name to something like `version-bump.sh` just to keep it semantically clear and unambiguous?